engines/stark/ui/world/gamescreen.cpp

```diff
@@ -62,12 +64,15 @@ GameScreen::~GameScreen() {
 }
 
 void GameScreen::open() {
+	if (g_engine->isPaused()) {
+		g_engine->pauseEngine(false);
+	}
 	StarkUserInterface->freeGameScreenThumbnail();
 }
 
 void GameScreen::close() {
 	_cursor->setMouseHint("");
-	StarkUserInterface->saveGameScreenThumbnail();
+	g_engine->pauseEngine(true);
```

Comment:
Pausing the engine while the game menu is open is not ok. It prevents the GMM from working.
Only the game entities need to be paused, adding a pause event handler to the Sound entities to pause them in the mixer.